deltachat-jsonrpc/src/api/mod.rs

```diff
@@ -1520,6 +1520,42 @@ impl CommandApi {
             .context("path conversion to string failed")
     }
 
+    async fn misc_save_sticker(
+        &self,
+        account_id: u32,
+        message_id: u32,
+        collection: String,
+    ) -> Result<()> {
+        let ctx = self.get_context(account_id).await?;
+        let message = Message::load_from_db(&ctx, MsgId::new(message_id)).await?;
+        if message.get_viewtype() != Viewtype::Sticker {
+            bail!("message is not a sticker");
+        }
+        let account_folder = ctx
+            .get_dbfile()
+            .parent()
+            .context("account folder not found")?;
+        if collection.contains('/') || collection.contains('\\') || collection.contains('.') {
+            bail!("illegal characters in collection name");
```

Comment:
for now it is set by the UI to the translation of the "saved" - translation string. Later the idea is that desktop could offer to save it to existing folders.

If we want to get it from the network we'll quickly run into other questions that might require cryptography, like how to prevent a remixed sticker to be sorted into the "official" pack.